test/e2e/kubectl/kubectl.go

```diff
@@ -501,6 +501,16 @@ var _ = SIGDescribe("Kubectl client", func() {
 			}
 		})
 
+		// https://issues.k8s.io/128314
+		ginkgo.It("should support exec idle connections", framework.WithSlow(), func(ctx context.Context) {
```

Comment:
I gave you the wrong tag
```
Output from proc 1:
  Unknown Decorator
  k8s.io/kubernetes/test/e2e/kubectl/kubectl.go:505
    [It] node was passed an unknown decorator:
    'framework.label{parts:[]string{"Slow"}, extraFeature:"", explanation:"If you
    see this as part of an 'Unknown Decorator' error from Ginkgo, then you need to
    replace the ginkgo.It/Context/Describe call with the corresponding
    framework.It/Context/Describe or (if available) f.It/Context/Describe."}'
    Learn more at: http://onsi.github.io/ginkgo/#node-decorators-overview

```
```suggestion
		ginkgo.It("should support exec idle connections", f.WithSlow(), func(ctx context.Context) {
```

Reply:
Thanks. Updated.